### PR TITLE
Issue 2 user like dislike art

### PR DESCRIPTION
**Pull request by khan-osama**
https://user-images.githubusercontent.com/63980419/109367032-d8ca1480-7849-11eb-9d74-9b1b5eb42978.mp4

### CODE REVIEW

**Comment by khan-osama** [comment]
https://user-images.githubusercontent.com/63980419/109862989-f286a580-7c15-11eb-8254-65a8de7b2ec3.mp4

File: js/main.js [hunk 1]
```diff
@@ -64,11 +67,74 @@ function changeToAddArtPage(event) {
...
+  $homePageFull.className = 'home-page-filled';
+  $artList.className = 'art-container-list';
+});
+
+/* var observer = new MutationObserver(addedPlusIcons);
+
+function addedPlusIcons(event) {
+  var $addIcons = document.querySelectorAll('.add-icon');
+  console.log($addIcons);
+} */
```
Comment (codingcodymiller) [inline js/main.js]: Remove commented code before submitting

File: js/main.js [hunk 2]
```diff
@@ -64,11 +67,74 @@ function changeToAddArtPage(event) {
...
+  var $addIcons = document.querySelectorAll('.add-icon');
+  console.log($addIcons);
+  $addIcons.forEach(addButton => {
+    addButton.addEventListener('click', event => {
+      if (addButton.className === 'add-icon') {
+        addButton.className = 'remove-icon';
+        addButton.setAttribute('src', 'images/minus-sign.svg');
+        var selectedParent = addButton.parentElement;
+        var selectedGrandParent = selectedParent.parentElement;
+        /* data.likedImages.push(selectedGrandParent); */
```
Comment (codingcodymiller) [inline js/main.js]: Remove commented code before submitting

File: js/main.js [hunk 3]
```diff
@@ -64,11 +67,74 @@ function changeToAddArtPage(event) {
   $divPage3.className = 'hidden confirmation-page';
   $divPage4.className = 'page4';
   data.view = 'add-art-page';
+  var $addIcons = document.querySelectorAll('.add-icon');
+  console.log($addIcons);
+  $addIcons.forEach(addButton => {
+    addButton.addEventListener('click', event => {
```
Comment (codingcodymiller) [inline js/main.js]: We haven't covered arrow functions yet, is this possible without them?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**